login-workflow/docs-new/language-support.md

```diff
@@ -8,6 +8,39 @@ This package supports translations to different languages using [i18next](https:
 -   Spanish
 -   Simplified Chinese
 
-## Changing the Language
+## Add/Override translations within our workflow
 
-<!-- @TODO: Finish this... -->
\ No newline at end of file
+To add i18n in your application you can follow the instructions from [official website](https://react.i18next.com/getting-started).
+You need to wrap your app with <I18nextProvider/> for rendering translations in the app. For reference, you can check [I18nextProvider](https://react.i18next.com/latest/i18nextprovider).
+
+
+If you want to add/override your custom screens translations in our `Auth Workflow` you need to pass your app's i18n instance to the `AuthContextProvider` via the `i18n` prop.
```

Comment:
Should the doc reference anything from both RegistrationContextProvider & AuthContextProvider dictionaries or shared dictionaries? How to actually add additional languages via the 2 providers. Just asking because its not clear how an end user would actually add more languages. Or will this be covered somewhere else...

Reply:
Here we have mentioned we are supporting only 5 languages https://github.com/etn-ccis/blui-react-workflows/blob/89180d38b6c35e970878bc725f1eaea64dce5fd7/login-workflow/docs-new/language-support.md?plain=1#L3. I believe if the user passes any different language `app language` will change but our workflows translations will break.

Reply:
We need to be able to support users adding additional dictionaries/languages for the internal dictionaries

Reply:
Done. please review.